README.md

```diff
@@ -0,0 +1,111 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Final Capstone Front-end
+This is the front end of the final capstone.
+The user can signup or login, then it will be redirected to the home page after a successful login. The user can see all the models of bikes on the page, after clicking them the user can add a reservasion to a specific bike and if the user wants to add a reservasion without going through this procces it can be done by a link on the navbar. Additionally, the user can see their reservations, and create or delete a bike.
+<hr>
+
+## Live demo
+
+- [Ducati Rentals](https://ducati-rentals.netlify.app/)
+<hr>
+
+## Api
+This app is done with a backend developed by our team.
+- [Demo](https://bikes-db.onrender.com/api-docs/index.html) 
+- [Source code](https://github.com/AbrahaKahsay/final-capstone-backend) 
+## Built With
+ ![VSCode](https://img.shields.io/badge/VSCode-0078D4?style=for-the-badge&logo=visual%20studio%20code&logoColor=white)
+ ![React](https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB)
+ ![Redux](https://img.shields.io/badge/Redux-593D88?style=for-the-badge&logo=redux&logoColor=white)
+-  React.js
+-  Redux
+-  JavaScript
+-  Html
+-  Linters: Eslint
+-  Linters: [Stylent](https://stylelint.io/)
+-  Visual Studio Code (code editor)
+<hr>
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+-  A code editor like Visual Studio Code with Git and Node.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for node installation.
+```
+$ node --version
+```
+
+### Install
+
+    In the terminal, navigate to where you wish the repo to be located and run these commands.
+
+```
+$ git clone https://github.com/AbrahaKahsay/final-capstone-frontend.git
+```
+```
+$ cd final-capstone-backend
```

Comment:
- [ ] This dir should be final-capstone-frontend if we are trying to setup the frontend to run.  Kindly correct it.

Reply:
Will do thanks